tests/datasets/test_mmearth.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import shutil
+from itertools import product
+from pathlib import Path
+
+import pytest
+import torch
+import torch.nn as nn
+from _pytest.fixtures import SubRequest
+
+from torchgeo.datasets import DatasetNotFoundError, MMEarth
+
+pytest.importorskip('h5py', minversion='3.6')
+
+data_dir_dict = {
+    'MMEarth': 'tests/data/mmearth/data_1M_v001/',
+    'MMEarth64': 'tests/data/mmearth/data_1M_v001_64/',
+    'MMEarth100k': 'tests/data/mmearth/data_100k_v001/',
```

Comment:
Do these need os.path.join so it uses \ on Windows? Or does `Path` handle that internally?